WCON_format.md

```diff
@@ -280,12 +283,10 @@ An example WCON file with a complete metadata section is given below.
            "media":"NGM",
            "sex":"hermaphrodite",
            "stage":"dauer",
-           "age":38.4,
+           "age":18.511,
```

Comment:
Why did you change this?  38.4 is a reasonable age (in hours) to get a dauer larva due to overcrowding.  Do you know a protocol that will give you dauers in 18.5 hours?  (High temperature?)